core/src/test/java/tech/pegasys/web3signer/core/util/OpenApiSpecsExtractorTest.java

```diff
@@ -13,43 +13,26 @@
 package tech.pegasys.web3signer.core.util;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatCode;
-
-import tech.pegasys.web3signer.core.Runner;
 
 import java.io.IOException;
 import java.io.UncheckedIOException;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Optional;
 
-import io.vertx.core.Vertx;
-import io.vertx.junit5.VertxExtension;
-import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 
-@ExtendWith(VertxExtension.class)
 class OpenApiSpecsExtractorTest {
 
   @ParameterizedTest
   @ValueSource(
       strings = {"eth1/web3signer.yaml", "eth2/web3signer.yaml", "filecoin/web3signer.yaml"})
-  void openapiSpecsAreExtractedAndLoaded(final String spec, final Vertx vertx) throws Exception {
+  void openapiSpecsAreExtractedAndLoaded(final String spec) throws Exception {
```

Comment:
I think we should be able to remove OpenApiSpecsExtractor class and its usage as the whole reason was to modify the paths in the openapi spec so that it is readable by the parser.

Reply:
Wasn't able to remove it, it's still needed to copy the files from the jar to the filesystem so we can serve the static content for the swagger UI